docs/gettingstarted/installation.rst

```diff
@@ -260,21 +260,19 @@ You'll probably want to setup at least two nodes.
 #. Install a supported Linux kernel
 
    Kernels older than ``3.16.4`` have a bug that affects Flocker's use of ZFS.
-   At the time of writing, the only supported Fedora 20 kernel on DigitalOcean is ``Fedora 20 x64 vmlinuz-3.16.6-203.fc20.x86_64``.
-   To switch to that kernel, follow these steps:
+   To switch to the newest kernel, follow these steps:
 
-   #. Upgrade the kernel package inside the virtual machine:
-
-      This specific kernel is no-longer available from the standard Fedora 20 repositories, so we install from the ``kojipkgs`` repository directly.
+   #. Configure the Droplet to boot with the desired kernel:
 
-      .. code-block:: sh
+      * Go to the DigitalOcean control panel for your specific Droplet, and in the Settings section choose the Kernel tab.
+      * Choose the newest kernel for Fedora 20 (scroll all the way to the bottom) and press "Change".
```

Comment:
Note to self: Try defining the kernel URL / version using:
 * http://sphinx.readthedocs.org/en/latest/config.html#confval-rst_epilog